sos/collector/clusters/ocp.py

```diff
@@ -152,6 +188,16 @@ def cleanup(self):
         """Remove the project we created to execute within
         """
         if self.project:
+            # cleanup cluster role binding
+            if os.path.exists(self.rbac_tmp):
+                os.unlink(self.rbac_tmp)
+            ret = self.exec_primary_cmd(
+                self.fmt_oc_cmd("delete clusterrolebinding %s"
+                                % self.rbac_name))
+            if ret['status'] != 0 or "deleted" not in ret['output']:
+                self.log_debug("Calling delete on clusterrolebinding '%s' "
+                               "failed: %s" % (self.rbac_name, ret['output']))
+                return False
```

Comment:
If we exit here, we won't delete the project, which could cause issues for subsequent runs. Shouldn't we always (attempt to) delete the project at the end of a collection?

Also, while the existing debug message is fine so that we can dig into a particular failure, I think a user-level error notification would be helpful. It doesn't have to specify the reason, a simple "failed to delete sos clusterrolebinding" or similar would suffice.